data/human/culture conversations.txt

```diff
@@ -37,6 +37,21 @@ mission "Sad Gamblers - Syndicate"
 
 
 
+mission "Foiled Kidnapping Witnessed"
+	minor
+	source
+		government "Syndicate"
+		not attributes "station" "frontier" "urban"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`A roadblock interrupts your path as you make your way towards the city, forcing you to slow down as you pass by, able to see what the commotion is about. Police wave past car after car coming from the other direction, looking into the vehicles until a sedan gets pulled over to the side causing police to briefly halt all traffic movements. The driver seemed distressed, refusing to open the boot even when he is being detained; his face goes pale when a little girl is found inside.`
```

Comment:
```suggestion
			`A roadblock interrupts your path as you make your way towards the city, forcing you to slow down as you pass by, able to see what the commotion is about. Police wave past car after car coming from the other direction, looking into the vehicles until a sedan gets pulled over to the side, causing police to briefly halt all traffic movements. The driver seems distressed, refusing to open the boot even when he is being detained; his face goes pale when a little girl is found inside.`
```

Reply:
```suggestion
			`A roadblock interrupts your path as you make your way towards the city, forcing you to slow down as you pass by, able to see what the commotion is about. Police wave past car after car coming from the other direction, looking into the vehicles until one gets pulled over to the side, causing police to briefly halt all traffic movements. The driver seems distressed, refusing to open the back even when he is being detained; his face goes pale when a little girl is found inside.`
```
Reading `sedan` feels awfully odd in a game set 1000 years into the future to me. Also, `boot` is British English for what North Americans would call the `trunk`. I think `back` is a reasonable compromise of the two.

Reply:
Yep. We'll use the compromise.